docs/resources/cluster.md

```diff
@@ -646,7 +646,7 @@ The following attributes are exported:
 * `enable_cri_dockerd` - (Optional) Enable/disable using cri-dockerd. Deafult: `false` [enable_cri_dockerd](https://rancher.com/docs/rke/latest/en/config-options/#cri-dockerd) (bool)
 * `ignore_docker_version` - (Optional) Ignore docker version. Default `true` (bool)
 * `ingress` - (Optional/Computed) Kubernetes ingress configuration (list maxitems:1)
-* `kubernetes_version` - (Optional/Computed) K8s version to deploy. Default: `Rancher default` (string)
+* `kubernetes_version` - (Optional/Computed) K8s version to deploy. Default: `Rancher default` (string) (Note - if rke_config is set, kubernetes_version must be set to the active cluster version so Rancher can clone the RKE template)
```

Comment:
The added note is true just if `rke_config` is set from `cluster_template`, isn't it? May be it should be more explicit. 

`(Note - if rke_config is set at cluster_template, kubernetes_version must be set to the active cluster version so Rancher can clone the RKE template)`